cinn/auto_schedule/database/database.h

```diff
@@ -34,8 +40,34 @@ struct TuningRecord {
   struct Compare {
     bool operator()(const TuningRecord& lhs, const TuningRecord& rhs) const;
   };
+
+  // TuningRecord(const proto::TuningRecord& record_proto)
+  //           : task_key(record_proto.task_key()),
+  //             execution_cost(record_proto.execution_cost()),
+  //             state(ir::ModuleExpr()) {}
+
+  // convert to proto object
+  proto::TuningRecord ToProto() const;
+  // convert to string in JSON format
+  std::string ToJSON() const;
 };
 
+// create a TuningRecord object from a proto object
+static std::shared_ptr<TuningRecord> BuildTuningRecord(const proto::TuningRecord& record_proto) {
+  std::shared_ptr<TuningRecord> record(
+      new TuningRecord({record_proto.task_key(), record_proto.execution_cost(), SearchState(ir::ModuleExpr())}));
+  return record;
+}
```

Comment:
添加一个TuningRecord构造函数实现从Proto解析构造，并保留默认构造
```suggestion
TuningRecord(proto::TuningRecord& record_proto);
TuningRecord() = default;
```

Reply:
> 添加一个TuningRecord构造函数实现从Proto解析构造，并保留默认构造

Done